third_party/s2n-bignum/include/s2n-bignum_aws-lc.h

```diff
@@ -15,6 +15,14 @@
 #ifndef S2N_BIGNUM_AWS_LC_H
 #define S2N_BIGNUM_AWS_LC_H
 
+#include <openssl/target.h>
+
+#if defined(OPENSSL_WINDOWS)
```

Comment:
Is the issue Windows or MSVC? Do other compilers on Windows support static?

Reply:
Okay, adopted the approach I implemented for https://github.com/awslabs/s2n-bignum/pull/98.